app/my_script.py

```diff
@@ -1,4 +1,107 @@
-import os
-import pwd
+import json
+import logging
+import pandas as pd
 
-print("Hello {} - welcome to my template-world!".format(pwd.getpwuid(os.getuid())[0]))
+# Initialize log
+log = logging.getLogger(__name__)
+
+DLR_MRID_FILEPATH = '/home/tsp/github/conduck/tests/valid-testdata/DLR_MRID.csv'
+MRID_KEY_NAME = 'terminal_name'
+MRID_VALUE_NAME = 'segment_id'
+MRID_EXPECTED_HEADER_NAMES = ['terminal_name', 'far_near', 'measurement_id', 'segment_id', 'dlr_enable']
+
+LIMITS_NAME_FILEPATH = '/home/tsp/github/conduck/tests/valid-testdata/Limits_other.xlsx'
+DD20_DATASHEET_NAME = 'DD20Mapping'
+DD20_KEY_NAME = 'DD20 Name'
+DD20_VALUE_NAME = 'ETS Name'
+
+
+def parse_csv_file_to_dataframe(file_path: str, header_index: int = 0) -> pd.DataFrame:
+    '''
+    Read CSV file and parse it to pandas dataframe.
+    Parameters
+    ----------
+    file_path: str
+        Full path of the excel file.
+    header_index: int
+        Index number for row to be used as header (Default = 0)
+    Returns
+    -------
+        pd.DataFrame
+            A dataframe containing the data from csv file.
+    '''
+    # Trying to read data from CSV file and convert it to a dataframe.
+    try:
+        dataframe = pd.read_csv(file_path, delim_whitespace=True, on_bad_lines='skip',
+                                header=header_index)
+        dataframe.drop(dataframe.head(1).index, inplace=True)
+        log.info(f'CSV data in "{file_path}" was parsed to dataframe.')
+    except Exception as e:
+        log.exception(f'Parsing data from: "{file_path}" failed with message: {e}.')
+        raise
+
+    return dataframe
+
+
+def define_dictonary_from_two_columns_in_a_dataframe(dataframe: pd.DataFrame, dict_key: str,
+                                                     dict_value: str) -> dict:
+    '''
+    Read dataframe and parse it to dictonary.
+    Parameters
+    ----------
+    dataframe: pd.DataFrame
+        Dataframe to convert.
+    dict_key: str
+        Column name to be used as key for the dictonary
+    dict_value: str
+        Column name to be used as value for the dictonary
+    Returns
+    -------
+        dict
+            A dictionary with the key/value specified by the user.
+    '''
+    # Checking the dictonary key and value to ensure that the user input is found in the dataframe.
+    if dict_key not in dataframe:
+        log.exception(f'The column "{dict_key}" is not found in the dataframe, check "dict_key" input to the function.')
+        raise
```

Comment:
```suggestion
        raise KeyError(f'The column "{dict_key}" is not found in the dataframe, check "dict_key" input to the function.')
```
If not part of a try/except, please specify error.

Reply:
I wil use sys.exit(1) for now but will look into raise